### ISSUE TITLE
Design of the top page

### ISSUE THREAD

**Issue by kotarohara** [ProjectSidewalk/SidewalkWebpage#32]
Matt's comment: First page, looks nice but I don’t really know what the little indicators under the map mean… is this map supposed to be interactive? Okay, sometimes it is… when I hover over the right spots, but it doesn’t do anything which is something I sort of expected. The description, or call to action, leaves me asking what accessibility problems? And, what am I actually getting involved with? Some kind of image carousel instead of the map that explains the project visually and what I need to do might be helpful.

**Comment by kotarohara** [comment]
I'll come back to the design of the top page when I'm done with fixing the issues for the labeling interface.

**Comment by nguyenist** [comment]
@kotarohara label here is high priority. shall i work on this one next? i know you already have a ton of ideas. if so, let me know what you're thinking. if not, happy to provide a first pass at how to make the landing page better. my gut says remove the landing page and go directly into https://sidewalk.umiacs.umd.edu/audit

**Comment by kotarohara** [comment]
@nguyenist Hmmm, I would like to have a landing page, which allows us to give more context to people visiting the site. I'll actually have design prototypes that I will go through with people in the lab, so let me handle this :)

**Comment by nguyenist** [comment]
Sweet!

**Comment by kotarohara** [comment]
Note to myself: I should make a placeholder image for the top-page video.

**Comment by jonfroehlich** [comment]
@nguyenist It would be great to get your feedback on the landing page redesign. Kotaro did a very nice job of creating a number of nice alternative designs. We consider it one of *the* most important parts of our website as it should: (i) inform and excite people about our mission; (ii) motivate them to donate their time and work; (iii) look good!

**Comment by jonfroehlich** [comment]
And the landing page is currently in flux it looks like. @kotarohara pushed a new version but it is not rendering correctly, at least on my 30" monitor.

![image](https://cloud.githubusercontent.com/assets/1621749/14624719/33004058-05ab-11e6-8dfe-ee09f3f572f6.png)

**Comment by kotarohara** [comment]
@jonfroehlich I think it's cache.

**Comment by kotarohara** [comment]
@nguyenist Yes, it would be great if you could provide feedback as @jonfroehlich said.

I'm changing this from a `high priority` issue to `discussion` as we are done with the first version.

**Comment by jonfroehlich** [comment]
So, the landing page is greatly improving. Some comments.

1. For aesthetic reasons, I think we should take out the menu bar at top. If we do this, then we need two buttons: "Participate" and "Sign-In"
![image](https://cloud.githubusercontent.com/assets/1621749/14649710/4def4eaa-0636-11e6-953b-226349c4264b.png)

2. I think we need to darken the opacity overlay on the video. I think we should also reduce the video dimensions, so that we can better see what's going on. As now, it's really hard to see what's going on--the interfaces are cutoff/cropped.

3. Some of the video has whitespace on left and right side, which makes it look weird. We should have fullscreen video.

![image](https://cloud.githubusercontent.com/assets/1621749/14649831/c5c76f8e-0636-11e6-8724-aa481d1fc17d.png)

**Comment by jonfroehlich** [comment]
The video size is fine on my 30" monitor, so I think we just need to scale things differently, perhaps, for smaller monitors/resolutions.

![image](https://cloud.githubusercontent.com/assets/1621749/14651195/dd025f50-063c-11e6-8762-67e4e42afeaf.png)

**Comment by nguyenist** [comment]
Hey @kotarohara wanted to work on this but the site is down (https://sidewalk.umiacs.umd.edu/audit). Assuming you're working on it or is there a different link? Let me know if I should wait to provide comments / edits on this one.

**Comment by jonfroehlich** [comment]
Looks like it's still down... :(

**Comment by nguyenist** [comment]
Some aesthetic suggestions:
1. Update the icons. The previous ones had inconsistent line widths and therefore looked less clean aesthetically than it could have been. Hope you like them, if not-- don't feel like you have to use them.
2. I also changed the formatting a bit so it looked less squished together. Pull request soon.

![image](https://cloud.githubusercontent.com/assets/4484141/14789782/20a19336-0adc-11e6-8124-90259a28906e.png)

**Comment by jonfroehlich** [comment]
I think it's also crucial that we mention the computer vision part of this project--that is, how the labels will be used not just for accessibility aware GIS apps but to train CV algorithms to make labeling even faster. Telling this story, I think, is absolutely critical to portraying our application and to building excitement. I can work with you on the direct language... but I think we should literally have a very simple flow diagram that shows how labels are used to help train CV algorithms.

**Comment by nguyenist** [comment]
Ok I'll hold off on the pull request. Let's work on exact phrasing for these points:
- Use our tool to label accessibility attributes in Google Street View
- The collected data will be stored on our server
- [Your labels] will be used to train our algorithms to make labeling (for what specifically) faster?
- Accessibility will be visualized with the collected data -- I think we need to reword this to read a little smoother. Perhaps: Accessibility [routes? maps? visualizations?-- would be helpful to have a specific noun here] can be created with collected data

@jonfroehlich Would you like to add the flow diagram as another icon above? Or is this a separate visualization?

**Comment by nguyenist** [comment]
Also, there are other suggestions I have for the home page, but would like to start small and touch base with you all to see where you are with opportunities for improvement so we can prioritize the most important thing for your team!

**Comment by kotarohara** [comment]
**Icons and other design features** (@nguyenist)
I like the icons you designed. It's cleaner than the ones I made/used; they were too dark and didn't match the overall aesthetics. Can you make the versions without the outer circle for each icon though? I might like them even better. It would be great if you could paste the screenshot(s) of versions of the flowcharts like you did previously. 

Also, do you want to take over the task of improving the design of the landing page?

**Information about the automation algorithms** (@jonfroehlich @nguyenist)
I agree we should mention it somewhere on the landing page. But, I disagree that it should be integrated into the current flowchart. The flowchart should briefly and quickly inform what the task is for the people who are willing to contribute and how the data is used. Adding additional information will dilute that message. 

Alternatively, we could have sections below the flowchart, each of them detailing corresponding stage of the flowchart. Like: (1) first section describing how we gather data manually, (2) the second section telling a story about the fact that we will store and distribute accessibility data, and (3) we use the collected data for multiple different goals, including making applications like accessibility geo-visualization, a11y-aware navigation, etc, and how we use the collected data to train CV algorithms. Obviously, one downside of this structure is that the CV component comes at the bottom of the page, but I think it's not a big problem.

**Web page down** (@ngyenist)
Thanks for pointing that out! Yes, I'm aware. Our server shut down for maintenance and killed the application process. I restarted the process and [I think I fixed the issue.](https://github.com/ProjectSidewalk/Instructions#automatically-restarting-the-web-applications) I wrote a script that reacts to server reboot and runs the application. I hope it works, but I can't really test it... (I don't think the admin people will shutdown the system for me just to test if one script is working.)

**Comment by nguyenist** [comment]
**Icons and other design features**
Yes I will take on improving the landing page!

**Information about the automation algorithms**
Need more context on this to weigh in.

Here are some edits: icons, formatting, text consistency: https://github.com/ProjectSidewalk/SidewalkWebpage/pull/90/commits/cad7268a7db0ed09e3154d86af4f4dafbb8e5e5b

Attaching the images for you here also if helpful:
![cloudstore](https://cloud.githubusercontent.com/assets/4484141/14804330/3ea6a552-0b30-11e6-998c-394d00bf8f0a.png)
![labelinginterface](https://cloud.githubusercontent.com/assets/4484141/14804331/3ea6ba10-0b30-11e6-82f1-13027a585bf1.png)
![map](https://cloud.githubusercontent.com/assets/4484141/14804332/3ea8127a-0b30-11e6-9fe1-9bc6a0cc0018.png)

**Comment by jonfroehlich** [comment]
@kotarohara I think we need to discuss more whether CV algorithms should be integrated into the default flow chart. I vote yes. The notion that these labels are not just used to build apps but to actually train computer vision (i.e., machine learning) algorithms to speed up the process is both fascinating and motivating. My vote is that this is a core part of our story and thus should be part of the flowchart. I feel pretty strongly about this. 

The beauty of our work is that we are essentially double dipping on human labor. We need to express that.

**Comment by nguyenist** [comment]
Helpful research for me and thought I'd share / document. Looking at landing pages that relate to crowdsourcing something. Specifically trying to focus on design elements that show: (1) how they best explain what they do (2) how they explain the value of a participant's work. 
- http://pulse.media.mit.edu/
- http://climatecolab.org/
- https://www.waze.com/
- http://www.tiramisutransit.com/#reloaded
- https://openideo.com/
- https://www.mturk.com/mturk/welcome
- https://www.yelp.com/writeareview/biz/WW3wAnODM6bZZlnlIqV2TQ?return_url=%2Fbiz%2FWW3wAnODM6bZZlnlIqV2TQ

**Comment by nguyenist** [comment]
@kotarohara An idea. Would love your thoughts. What could be helpful is to see the tangible outcomes of helping to label accessibility points visually. 

Currently you use these 2 paragraphs. I think we should use less text, more imagery to condense these concepts. Design for ease of scanning content and assume not everyone will read all text in depth. 

Instead of this:
```
We need your help to map the accessibility of the city. Problems such as lack of curb ramps make navigation more difficult for people with mobility impairments. Knowing where problems exist will help people plan their trip. Use our tool to walk through streets in Google Street View and report where the accessibility problems exist. We will use the data to visualize the city’s accessibility!

The collected accessibility data could be used in accessibility-aware map applications. For example, we could visualize whether one neighborhood has problems or not. The visualization allows a person with mobility impairment (e.g., a wheelchair user) to decide if they can go to their destination alone or they need someone to tag along to help them navigate inaccessible sidewalks.
```
Right now you subtly allude to it, but I think this should be more obvious. Show the *value* of your data in a very concrete / tangible way. For example (I made these up but know you probably have a better grasp of the value proposition). 

With your help we can use the data to:
1. Create accessibility-aware maps so people can plan their travel. Ex: visualize whether one neighborhood has problems or not. 
2. Improve city safety. Provide city governments with data to include planning and development opportunities to improve their neighborhood streets. 
3. Increase academic awareness. For civic based research on city accessibility to continue initiatives to make sidewalks more walkable.

See the placemeter.com example here. 
![image](https://cloud.githubusercontent.com/assets/4484141/14823028/f56a2264-0b9e-11e6-9d57-64e90900dfa0.png)

**Comment by kotarohara** [comment]
Addressed computer vision thing. Added a text saying "The collected accessibility data will also be used to train computers to automatically find and label accessibility attributes at scale and fast."

**Comment by kotarohara** [comment]
Got rid of the demo video. Because the video we had was designed as a background video, it doesn't have any textual explanation of what's going on in the shot and doesn't convey what's going on to the viewers. Maybe put it back in the future if we make a good demo video.

With a video:
![cursor_and_project_sidewalk](https://cloud.githubusercontent.com/assets/612263/15848020/3680299a-2c58-11e6-8c19-0d516ece698f.png)


Without a video:
![cursor_and_project_sidewalk 2](https://cloud.githubusercontent.com/assets/612263/15847926/abe70164-2c57-11e6-9180-ea5169ca31e1.png)

**Comment by nguyenist** [comment]
@kotarohara did you happen to get any user feedback on the video?

**Comment by kotarohara** [comment]
@nguyenist No, we need to get feedback and iterate on it. The video issue moved to #107
